packages/fuels-types/src/core.rs

```diff
@@ -0,0 +1,119 @@
+use crate::{enum_variants::EnumVariants, errors::CodecError, param_types::ParamType};
+
+use fuel_types::bytes::padded_len;
+use strum_macros::EnumString;
+
+use std::fmt;
+
+pub mod bits;
+pub mod byte;
+pub mod native;
+pub mod sized_ascii_string;
```

Comment:
I wouldn't make this public. 
These types are split into files, not as some grand organization plan but rather to limit file size to something manageable. 

This doesn't need to be seen outside, especially since you're exporting everything a few lines below.